cmd/lncli/commands.go

```diff
@@ -373,11 +374,32 @@ func sendCoins(ctx *cli.Context) error {
 			"sweep all coins out of the wallet")
 	}
 
+	client, cleanUp := getClient(ctx)
+	defer cleanUp()
+	minConfs := int32(ctx.Uint64("min_confs"))
+
+	// In case that the user has specified the sweepall flag, we'll
+	// calculate the amount to send based on the current wallet balance.
+	displayAmt = amt
+	if ctx.Bool("sweepall") {
+		balanceResponse, err := client.WalletBalance(
+			ctxc, &lnrpc.WalletBalanceRequest{
+				MinConfs: minConfs,
+			},
+		)
+		if err != nil {
+			return fmt.Errorf(
```

Comment:
nit: calls to `fmt.Errorf()` should actually use the more compact formatting exception documented here: https://github.com/lightningnetwork/lnd/blob/master/docs/code_formatting_rules.md#exception-for-log-and-error-message-formatting

Reply:
Oh, my bad, thanks! Fixed :)